src/main/java/com/practice/PRACTICE/service/EmpServiceImpl.java

```diff
@@ -13,6 +14,8 @@
 public class EmpServiceImpl implements EmpService{
     @Autowired
     private EmpRepo empRepo;
+    @Autowired
+    private Constants constants;
```

Comment:
The Class constant should be static so that you do not need to create any bean or dependency to the other classes.